src/components/filter/HotelId/NewHotels.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { useParams } from "react-router-dom";
+import HotelRoute from "./RouterHotel";
+//import HotelItem from "../../HotelItem";
+//import AvailableHotels from "../AvailableHotels";
+import nightIcon from '../../../img/icons/Night.png';
+import account from '../../../img/icons/Vector.png';
+import logo_vector from '../../../img/icons/logo_vector.png'
+
+const HotelDetail = () => {
+    const {id} = useParams()
+    const thisHotels = HotelRoute.find(prod => prod.id === id)
+
+
+    return (
+        <>
+        <nav className="stays animated-header-line"><a href="#" className="header-link">Stays</a></nav>
+        <nav className="attractions animated-header-line"><a href="#" className="header-link">Attractions</a></nav>
+        <div className="top-icon-night" style={{ backgroundImage: `url('${nightIcon}')` }} ></div>
+        <div className="top-icon-account" style={{ backgroundImage: `url('${account}')` }} ></div>
+        <nav className="top-icon-burger"></nav>  
```

Comment:
you don't need to copy your header. You should have reusable "Header" component, which is the same for the pages.
![image](https://user-images.githubusercontent.com/20189313/149682259-e4f91c5e-2b2e-4e1d-9686-a5eb3630f552.png)
or move the header and footer outside the routes as shown in the picture. This will be a good option